plugin/hover.py

```diff
@@ -420,3 +426,8 @@ def _update_views_async(self, enable: bool) -> None:
                     session_view.view.settings().set(SHOW_DEFINITIONS_KEY, False)
                 else:
                     session_view.reset_show_definitions()
+
+
+class LspCopyTextCommand(sublime_plugin.TextCommand):
+    def run(self, edit, text: str) -> None:
+        sublime.set_clipboard(text)
```

Comment:
Should update status with number of characters copied like the original copy does?

Reply:
[1d8a53a](https://github.com/sublimelsp/LSP/pull/2618/commits/1d8a53a732da98121fd6e15d41d0880bf895176f)